test/benchmarks/driverBench/index.js

```diff
@@ -89,15 +90,22 @@ benchmarkRunner
       return {
         info: {
           test_name: benchmarkName,
-          tags: [bsonType]
+          tags: [bsonType],
+          // Args can only be a map of string -> int32. So if its a number leave it be,
+          // if it is anything else test for truthiness and set to 1 or 0.
+          args: Object.fromEntries(
+            Object.entries(MONGODB_CLIENT_OPTIONS).map(([key, value]) => [
+              key,
+              typeof value === 'number' ? value | 0 : value ? 1 : 0
+            ])
+          )
         },
         metrics: [{ name: 'megabytes_per_second', value: result }]
       };
     });
   })
   .then(data => {
     const results = JSON.stringify(data, undefined, 2);
-    console.log(inspect(data, { depth: Infinity, colors: true }));
```

Comment:
I removed this because now that we print the results of the tests after each execution this _not_ printing this should make them all visible in the preview of the logs without having to open parsely (small DX ++)